src/libpgexporter/utils.c

```diff
@@ -888,10 +888,15 @@ pgexporter_set_proc_title(int argc, char** argv, char* s1, char* s2)
    max_process_title_size = size;
 
 #else
-   setproctitle("-pgexporter: %s%s%s",
-                s1 != NULL ? s1 : "",
-                s1 != NULL && s2 != NULL ? "/" : "",
-                s2 != NULL ? s2 : "");
+  #if defined(__linux__) || defined(__FreeBSD__) || defined(__NetBSD__) ||       \
```

Comment:
Please take a look at [this](https://www.ibm.com/docs/en/xl-c-and-cpp-linux/16.1.0?topic=macros-related-platform) for a complete list.